README.md

```diff
@@ -11,9 +11,8 @@ Our default implementation works with Amazon Web Services (AWS), but AWS Amplify
 *Note: Amplify docs are still being updated and will go live by EOW. The below links will take you to the SDK documentation currently.
 
 - [**Analytics**](https://aws-amplify.github.io/docs/ios/analytics): Easily collect analytics data for your app. Analytics data includes user sessions and other custom events that you want to track in your app.
-- [**API**](https://aws-amplify.github.io/docs/ios/api): Provides a simple solution when making HTTP requests. It provides an automatic, lightweight signing process which complies with AWS Signature Version 4.
-- [**GraphQL Client**](https://aws.github.io/aws-amplify/media/api_guide#configuration-for-graphql-server): Interact with your GraphQL server or AWS AppSync API with an easy-to-use & configured GraphQL client.
-- [**Storage**](https://aws-amplify.github.io/docs/ios/storage): Provides a simple mechanism for managing user content for your app in public, protected or private storage buckets.
+- [**API**](https://aws-amplify.github.io/docs/ios/api): Interact with your AWS AppSync API or make HTTP requests to your API Gateway endpoint with the Amplify API client. It provides a GraphQL client interface to use with Amplify's model generation and automatic signing process which complies with AWS Signature Version 4.
```

Comment:
The existing SDK docs are will be at https://aws-amplify.github.io/docs/sdk/ios/api once the docs are live today
Notice the "sdk"

Reply:
updated

Reply:
updating to the start guide for old SDK https://aws-amplify.github.io/docs/sdk/ios/start